src-built-in/components/userPreferences/src/stores/UserPreferencesStore.jsx

```diff
@@ -39,6 +40,22 @@ var Actions = {
         FSBL.Clients.WorkspaceClient.addWorkspaceTemplateDefinition(newTemplateJSONDefinition, { force: true }, function (err) {
             console.log("addWorkspaceTemplateDefinition result", err);
         });
+	},
+	/**
+	 * By making the value of `finsembke.scheduledRestart` falsy, the application will remove any existing restart timers.
+	 */
+	disableScheduledRestart: () => {
```

Comment:
@carterbs okay for now but we're going to need to circle back with a more generic way of displaying UI for preferences. We can't be asking developers to modify our store every time they want to add a preference.

@jimbunting this is technical debt.